README.md

```diff
@@ -1,40 +1,48 @@
-Moonstone [![NPM version](https://img.shields.io/npm/v/moonstone-wrapper.svg?style=flat-square&color=informational)](https://npmjs.com/package/moonstone-wrapper)
-====
+<!-- Made By Isaiah-Hamilton(https://github.com/Isaiah-Hamilton) -->
```

Comment:
Because I helped make the README.md. I also have a habit of adding my name and GitHub to my code. The reason is that I have had people steal my code line for line and say that they made it and no credit. But you can remove it if you want just as I said in my first commit.

> This README.md file shows people that have never made a doge bot or a bot in general how to make one. The images on the README.md will work on push to the main branch. You can get rid of the contributor's section if you want.

Reply:
Sure.